js/views/homeVw.js

```diff
@@ -129,11 +129,9 @@ module.exports = Backbone.View.extend({
     } else if(data.id == this.socketSearchID) {
       this.renderItem(data);
     }
-    this.resetLookingCount();
```

Comment:
Why did you take this out here? It should reset each time a new result comes in.